bxsolana/provider/ws.ts

```diff
@@ -128,7 +128,7 @@ import {
     GetRaydiumCLMMPoolsRequest,
     GetRaydiumCLMMPoolsResponse,
     GetRaydiumCLMMQuotesRequest,
-    GetRaydiumCLMMQuotesResponse,
+    GetRaydiumCLMMQuotesResponse, GetNewRaydiumPoolsByTransactionRequest, GetNewRaydiumPoolsByTransactionResponse,
```

Comment:
Should we have a test for GetPriorityFeeByProgram?

Reply:
I just omitted GetPriorityFeeByProgram because it will be included in my larger refactor